test/benchmarks/driverBench/common.js

```diff
@@ -67,7 +129,7 @@ function initCollection() {
 
 function dropCollection() {
   return this.collection.drop().catch(e => {
-    if (e.code !== MONGODB_ERROR_CODES.NamespaceNotFound) {
+    if (e.code !== 26 /* NamespaceNotFound */) {
```

Comment:
This isn't public API so testing arbitrary driver versions made this difficult when only importing at the package level.